extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -222,6 +227,37 @@ const BLOCK_SRA_COMBINERS_ = [
   },
 ];
 
+/**
+ * Used to manage messages for different fluid ad slots.
+ *
+ * Maps a sentinel value to an object consisting of the impl to which that
+ * sentinel value belongs and the corresponding message handler for that impl.
+ * @type{!Object<string, !{instance: !AmpAdNetworkDoubleclickImpl, connectionEstablished: boolean}>}
+ */
+const fluidListeners = {};
+
+/**
+ * @param {!Event} event
+ * @private
+ */
+function fluidMessageListener(event) {
```

Comment:
Private function name should end in underscore

Reply:
Done (thought that only applied to instance functions).